drivers/serial/uart_ns16550.c

```diff
@@ -950,12 +950,21 @@ static int uart_ns16550_line_ctrl_set(const struct device *dev,
 				      uint32_t ctrl, uint32_t val)
 {
 	struct uart_ns16550_dev_data *data = dev->data;
-	uint32_t mdc, chg;
+	const struct uart_ns16550_device_config *const dev_cfg = dev->config;
+	uint32_t mdc, chg, pclk = 0;
```

Comment:
If you start using `0U` below, probably you need to use it also here?

Reply:
Updated